src/feedhandler.cpp

```diff
@@ -5,9 +5,11 @@
 
 namespace newsboat {
 
-void FeedHandler::sort_feeds(const std::vector<std::string>& sortmethod_info)
+void FeedHandler::sort_feeds(configcontainer* cfg)
```

Comment:
*This* is what I meant when I said "[r]esist the urge to fix little things and improve interfaces "while you're at it". It's simply outside of the scope of the issue this PR addresses.

I opened #217 that'll fix this.